main.go

```diff
@@ -118,9 +119,10 @@ func main() {
 
 	grafeasExtensions := grafeas.NewExtensions(logger.Named("GrafeasExtensions"), grafeasClientCommon)
 	resourceManager := resource.NewManager(logger.Named("Resource Manager"), esutilClient, c.Elasticsearch, indexManager, filterer)
-	policyManager := policy.NewManager(logger.Named("PolicyManager"), esutilClient, c.Elasticsearch, indexManager, filterer, opaClient, grafeasExtensions)
+	policyManager := policy.NewManager(logger.Named("PolicyManager"), esutilClient, c.Elasticsearch, indexManager, filterer)
 	policyGroupManager := policy.NewPolicyGroupManager(logger.Named("PolicyGroupManager"), esutilClient, c.Elasticsearch, indexManager, filterer)
 	policyAssignmentManager := policy.NewAssignmentManager(logger.Named("PolicyAssignmentManager"), esutilClient, c.Elasticsearch, indexManager, filterer)
+	evaluationManager := evaluation.NewManager(logger.Named("EvaluationManager"), esutilClient, policyManager, policyGroupManager, policyAssignmentManager, grafeasExtensions, opaClient, resourceManager, indexManager)
```

Comment:
I really want to do something about this.  seems like every `Manager` interface we create inevitably depends on all of the others at some point.  maybe we should revisit using an actual DI framework